erpnext/public/js/utils.js

```diff
@@ -590,6 +591,20 @@ erpnext.utils.map_current_doc = function(opts) {
 	}
 }
 
+erpnext.utils.clear_duplicates = function() {
+	const unique_items = new Map();
+	let items = []
+
+	for (let i = 0; i < cur_frm.doc.items.length; i++) {
```

Comment:
```suggestion
	for (let item of cur_frm.doc.items) {
```

You can remove the `item` assignment on the next line as well.